app/locustio/bamboo/http_actions.py

```diff
@@ -44,49 +27,63 @@ def run_build_plans(locust):
     response = r.json()
     auth_headers = r.request.headers
     plan_is_active = response['isActive']
-    wait_for_online_free_agent()
 
-    @bamboo_measure('locust_run_build_plan')
     def run_build_plan(locust):
+        r = locust.post(f'/rest/api/latest/queue/{build_plan_id}', catch_response=True,
+                        headers=auth_headers, auth=user_auth)
+        build_num = r.json()['buildNumber']
+        build_job_num = f'{build_plan_id}-JB1-{build_num}'
```

Comment:
don't we have a job id from the response of line 32? If not, let's move `JB1` to variable at least.